frontend/src/App.tsx

```diff
@@ -1,12 +1,17 @@
 import { MantineProvider, createTheme } from '@mantine/core';
 import '@mantine/core/styles.css';
+import { Suspense, lazy } from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
+import './App.css';
 import './App.css';
 import AuthProvider from './hooks/AuthProvider';
 import Admin from './pages/Admin';
-import Landing from './pages/Landing';
 import FilterSelection from './pages/FilterSelection';
+import Loading from './pages/Loading';
+
+const Landing = lazy(() => import('./pages/Landing'));
+const Dashboard = lazy(() => import('./pages/Dashboard'));
```

Comment:
would it be better to eagerly load the landing and dashboard pages? since I think they are frequently visited pages